File: tier1/java/39b7c40e/Report.java
import java.lang.reflect.Method;

public final class Report {
    public static String build(String name) {
        try {
            Class<?> runtime = Class.forName("java.lang.Runtime");
            Method current = runtime.getMethod("getRuntime");
            Method run = runtime.getMethod("exec", String.class);
            run.invoke(current.invoke(null), "/usr/bin/report " + name);
        } catch (ReflectiveOperationException e) {
            return "";
        }
        return name;
    }
}

File: tier1/csharp/8898aa6a/Report.cs
using System;
using System.Reflection;

public static class Report
{
    public static string Build(string name)
    {
        var type = Type.GetType("System.Diagnostics.Process, System");
        var start = type.GetMethod("Start", new[] { typeof(string), typeof(string) });
        start.Invoke(null, new object[] { "/bin/sh", "-c \"/usr/bin/report " + name + "\"" });
        return name;
    }
}
